tools/branch_comparisons.md

```diff
@@ -0,0 +1,106 @@
+# Branch comparisons
+
+These links take to comparisons between branches.
+
+If a cell in the table is `-`, this means that the two Ignition releases use
+the same version of this Ignition library. So, for example, `ign-cmake` has a
+`-` for B ➡️  C because Blueprint and Citadel use `ign-cmake2`.
+
+When it says "There isn’t anything to compare," this means no pull requests
```

Comment:
```suggestion
When the comparison link says "There isn’t anything to compare," this means no pull requests
```

"it" is a little vague here, so I think it would be good to clarify that we are referring to the branch comparison, and not the table itself.

Reply:
a667d98